docs/components/base/two_wheeled_base.md

```diff
@@ -80,3 +80,14 @@ The following attributes are available for `two_wheeled_base` bases:
 ## Test the base
 
 {{< readfile "/static/include/components/test-control/base-control.md" >}}
+
+## Next steps
+
+To get started using your base, see the [base API](/appendix/apis/components/base/).
+For more configuration and development info, see
+
+{{< cards >}}
+{{% card link="/appendix/apis/components/base/" customTitle="Base API" noimage="true" %}}
```

Comment:
Why duplicate link to base API with the above sentence?

Reply:
Extra callout but I can remove the starting sentence